modules/flowable-event-registry/src/test/java/org/flowable/eventregistry/test/CustomEventProcessingPipelineTest.java

```diff
@@ -14,12 +14,13 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Map;
+import java.beans.FeatureDescriptor;
+import java.util.*;
```

Comment:
The project doesn't use wildcard imports. Please use the IDE settings provided in https://github.com/flowable/flowable-engine/tree/main/ide-settings